tests/fsharp/core/namespaces/test2.fs

```diff
@@ -13,7 +13,7 @@ module M =
 
 
 #if TESTS_AS_APP
-    let RUN() = !failures
+    let RUN() = !Hello.Goodbye.Utils.failures
```

Comment:
This was never working, even in the old test suite. (the TESTS_AS_APP code path was not being exercised)